src/App.tsx

```diff
@@ -1,39 +1,71 @@
-import React from 'react';
+import React, { useState, ChangeEvent } from 'react';
 import './App.scss';
-import Carousel from './components/Carousel';
-
-interface State {
-  images: string[];
-}
-
-class App extends React.Component<{}, State> {
-  state = {
-    images: [
-      './img/1.png',
-      './img/2.png',
-      './img/3.png',
-      './img/4.png',
-      './img/5.png',
-      './img/6.png',
-      './img/7.png',
-      './img/8.png',
-      './img/9.png',
-      './img/10.png',
-    ],
+import Carousel from './components/Carousel/Carousel';
+import Inputs from './components/Inputs/Inputs';
+
+const imagesLinks = [
+  './img/1.png',
+  './img/2.png',
+  './img/3.png',
+  './img/4.png',
+  './img/5.png',
+  './img/6.png',
+  './img/7.png',
+  './img/8.png',
+  './img/9.png',
+  './img/10.png',
+];
+
+const App: React.FC = () => {
+  const [itemWidth, setItemWidth] = useState(130);
+  const [frameSize, setFrameSize] = useState(3);
+  const [step, setStep] = useState(3);
+  const [animationDuration, setAnimationDuration] = useState(1000);
```

Comment:
It's not advisable to generate such a large number of states when we only have one instance (settings). 
It'd be much better to utilize a single state with an object inside.

1. Create a variable outside the Component
```
const initialSettingsState = {
  itemWidth: 130,
  frameSize: 3,
  step: 3,
  animationDuration: 1000,
}
```

2. Use one state for movie fields
3. Pass `initialSettingsState` object when you invoke reset `function`
4. No needs to use `switch`
Example: 

```
const handleInputChange = (e: ...) => {
  const { name, value } = e.target; 
  
  setSettings(prevSettings => ({...prevSettings, [name]: value}))
}
```

5. for reset function you will just pass initialMovieState


```suggestion
  const [settings, setSettings] = useState(initialSettingsState);
```